packages/commonwealth/client/scripts/hooks/useAppStatus.ts

```diff
@@ -0,0 +1,19 @@
+const useAppStatus = () => {
+  const isAddedToHomeScreen = window.matchMedia(
+    '(display-mode: standalone)',
+  ).matches;
+  const isMarketingPage = window.location.pathname === '/';
+  const isIOS = window.navigator.userAgent.match(/(iPad|iPhone|iPod)/g)
+    ? true
+    : false;
```

Comment:
This is a bad practice. 

`window.navigator.userAgent.match(/(iPad|iPhone|iPod)/g)` is a comparsion therefore returns a boolean. Lets get rid of the ternary here.

Reply:
It actually returns a RegExpMatchArray, which is why the ternary. Alternatively we can use !! to make sure it's boolean, but believed this is more verbose